Lib/idlelib/macosx.py

```diff
@@ -216,8 +216,6 @@ def help_dialog(event=None):
     root.bind('<<open-config-dialog>>', config_dialog)
     root.createcommand('::tk::mac::ShowPreferences', config_dialog)
     if flist:
-        root.bind('<<close-all-windows>>', flist.close_all_callback)
-
         # The binding above doesn't reliably work on all versions of Tk
         # on macOS. Adding command definition below does seem to do the
         # right thing for now.
```

Comment:
The comment “The binding above…” should likely also be adjusted so that it makes sense after removing the binding it refers to.